lumen/ai/tools.py

```diff
@@ -104,45 +104,100 @@ class TableLookup(VectorLookupTool):
     provides = param.List(default=["closest_tables"], readonly=True, doc="""
         List of context values this Tool provides to current working memory.""")
 
+    include_metadata = param.Boolean(default=True, doc="""
```

Comment:
I think the doc should be improved here. Does this include column descriptions too?